tests/test_gcp.py

```diff
@@ -39,7 +39,7 @@ class TestCallStageViaPubSub(unittest.TestCase):
 
```

Comment:
Given the amount of increase in service code base - do we need to implement more tests around it?

Reply:
I've added a test case for the service wrapper below